src/raglite/_eval.py

```diff
@@ -43,6 +41,9 @@ class QuestionResponse(BaseModel):
 - The question MUST treat the context as if its contents are entirely part of your working memory.
             """.strip()
 
+        class Config:
+            extra = "forbid"  # Ensure no extra fields are allowed as required by OpenAI API json schema.
+
```

Comment:
Is this also a 'strict' mode requirement?

Reply:
yes.